### PR TITLE
Adjust welcome email

### PR DESCRIPTION
**Pull request by znarf**
Fix: https://github.com/opencollective/opencollective/issues/3845

**Case 1 (No Host)**

Your {{accountType}} hasn't picked an Host yet. We now suggest you to find one on the platform. You will be able to collect money for your {{accountType}} without having to worry about accounting, invoicing, etc. If you already have a legal entity that could act as your host, please reach out!

**Case 2 (Host Pending)**

Your {{accountType}} is pending approval from your Host ({{host.name}}). Your Host is the legal entity that will hold the money on behalf of your {{accountType}}. Once they approve you, you will be able to collect money without having to worry about accounting, invoicing, etc.

**Case 3 (Host Approved)**

Your {{accountType}} is hosted by {{host.name}}. Your host is the legal entity that will hold the money on behalf of your {{accountType}}. With a Host, you will be able to collect money without having to worry about accounting, invoicing, etc.

### CODE REVIEW

File: templates/emails/collective.created.hbs [hunk 1]
```diff
@@ -4,14 +4,33 @@ Subject: Welcome to Open Collective!
...
+<p>I’m Pía, one of the founders of Open Collective. I’d love to help you set up and get your {{accountType}} up and running. Feel free to reach out to me directly by replying to this email. I'm here to help you!</p>
 
-<p>Your collective is pending approval from your host (<a href="https://opencollective.com/{{host.slug}}">{{host.name}}</a>). Your host is the legal entity that will hold the money on behalf of your collective. Once they approve you, you will be able to collect money for your collective without having to worry about accounting, invoicing, etc. If you already have a legal entity that could act as your host, please reach out!</p>
+{{#if host}}
+
+<p>
+  {{#if hostPending}}
+  Your {{accountType}} is pending approval from your Host (<a href="https://opencollective.com/{{host.slug}}">{{host.name}}</a>).
+  Your Host is the legal entity that will hold the money on behalf of your {{accountType}}.
+  Once they approve you, you will be able to collect money without having to worry about accounting, invoicing, etc.
```
Comment (piamancini) [inline templates/emails/collective.created.hbs:15]: (...) accounting, invoicing or tax reporting.

File: templates/emails/collective.created.hbs [hunk 2]
```diff
@@ -4,14 +4,33 @@ Subject: Welcome to Open Collective!
...
+  {{else}}
+  Your {{accountType}} is hosted by <a href="https://opencollective.com/{{host.slug}}">{{host.name}}</a>.
+  Your host is the legal entity that will hold the money on behalf of your {{accountType}}.
+  With a Host, you will be able to collect money without having to worry about accounting, invoicing, etc.
```
Comment (piamancini) [inline templates/emails/collective.created.hbs:19]: (...) accounting, invoicing or tax reporting.

File: templates/emails/collective.created.hbs [hunk 3]
```diff
@@ -4,14 +4,33 @@ Subject: Welcome to Open Collective!
...
+  {{/if}}
+</p>
+
+{{else}}
+
+<p>Your {{accountType}} hasn't picked an Host yet. We now suggest you to find one on the platform. You will be able to collect money for your {{accountType}} without having to worry about accounting, invoicing, etc. If you already have a legal entity that could act as your host, please reach out!
```
Comment (piamancini) [inline templates/emails/collective.created.hbs:25]: (...) doesn't have a Host yet. You can browse the hosts already one on the platform [here](https://opencollective.com/hosts). 

(...) accounting, invoicing or tax reporting. 

(...) act as your Host. Please select that option from your {{accountType}} settings in the tab, Fiscal Host. 

@znarf is my last edit a correct assumption here?

Comment (znarf) [reply templates/emails/collective.created.hbs:25]: @piamancini I don't think so. I just picked this sentence in the existing template. I think the idea was that if people had a Fiscal Host out of the platform (ie; Linux Foundation), they should reach out to see how we could onboard the Fiscal Host on the platform.

Comment (piamancini) [reply templates/emails/collective.created.hbs]: Oh I see, then it should be:
"If your collective is already under a Fiscal Host and you want them to join as well, please reach out!

**Pull request merged**